### PR TITLE
Akretion/9.0 migration account prod fiscal classif

### PR DESCRIPTION
**Pull request by mbcosta**

### CODE REVIEW

File: account_fiscal_position_rule/views/account_fiscal_position_rule_view.xml [hunk 1]
```diff
@@ -1,220 +1,231 @@
...
+				  parent="account.menu_finance_configuration" name="Fiscal Position Rule" sequence="21"/>
+
+		<!-- Fiscal Position Rule Template -->
+		<record id="view_account_fiscal_position_rule_template_form" model="ir.ui.view">
+			<field name="name">account.fiscal.position.rule.template.form</field>
+			<field name="model">account.fiscal.position.rule.template</field>
+			<field name="arch" type="xml">
+				<form string="Fiscal Position Rule Template">
+					<group>
+						<group string="General" colspan="4">
```
Comment (danimaribeiro) [inline account_fiscal_position_rule/views/account_fiscal_position_rule_view.xml]: Same goes here, add name="general" to the group.

File: account_product_fiscal_classification/__init__.py [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
```
Comment (legalsylvain) [inline account_product_fiscal_classification/__init__.py:22]: no need to set a big header for an init.py file.

**Comment by legalsylvain** [comment]
:+1: Thanks.

**Comment by rvalyi** [comment]
:+1: **init**.py header mention can be changed after merge, not a blocker for me.

**Pull request merged**